runtime/Cargo.toml

```diff
@@ -82,11 +82,11 @@ sp-transaction-pool = { git = 'https://github.com/paritytech/substrate', branch
 sp-version = { git = 'https://github.com/paritytech/substrate', branch = 'polkadot-v0.9.29', default-features = false }
 
 # Rmrk
-pallet-rmrk-core = { default-features = false, branch = "datahighway", git = "https://github.com/DataHighway-DHX/rmrk-substrate" }
-pallet-rmrk-equip = { default-features = false, branch = "datahighway", git = "https://github.com/DataHighway-DHX/rmrk-substrate" }
-pallet-rmrk-market = { default-features = false, branch = "datahighway", git = "https://github.com/DataHighway-DHX/rmrk-substrate" }
-rmrk-traits = { default-features = false, branch = "datahighway", git = "https://github.com/DataHighway-DHX/rmrk-substrate" }
-pallet-rmrk-rpc-runtime-api = { default-features = false, branch = "datahighway", git = "https://github.com/DataHighway-DHX/rmrk-substrate" }
+pallet-rmrk-core = { default-features = false, rev = "8c5a737", git = "https://github.com/DataHighway-DHX/rmrk-substrate" }
```

Comment:
Maybe it's better to leave it on branch versioning more than commit. It makes more sense that it points to the specific features on the `datahighway` branch, then on a commit hash.

Reply:
Agreed. SInce I am   on both repo in parallel, once commit is locked by cargo ( even when I specify branch, commit will be locked as per I understand ) it will not pull again the changes made. 

But for sure once PR is ready, it shall point to a branch